.remarkrc

```diff
@@ -0,0 +1,37 @@
+{
+  "plugins": [
+    ["remark-lint-no-dead-urls", ["error"]],
+    ["remark-lint-no-html", ["error"]],
+    [
+      "remark-frontmatter",
+      [
+        "yaml"
+      ]
+    ],
+    [
+      "remark-frontmatter-validator",
+      [
```

Comment:
Also it looks like we need to check for a Summary as it's shown in the search results